.licenserc.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright The async-profiler authors.
+# SPDX-License-Identifier: Apache-2.0
+
+header:
+  - paths:
+      - 'src/jattach/*'
+    license:
+      content: |
+        Copyright The jattach authors
+        SPDX-License-Identifier: Apache-2.0
+
+    comment: on-failure
+
+  - path: "**"
+    license:
+      content: |
+        Copyright The async-profiler authors
+        SPDX-License-Identifier: Apache-2.0
+
+    paths-ignore:
+      - 'Makefile'
+      - 'src/jattach/'
+      - 'pom.xml'
+      - 'pom-converter.xml'
+      - '.gitattributes'
+      - '.gitignore'
+      - '.github/'
+      - 'demo/'
+      - 'target/'
+      - 'src/res/'
+      - '**/*.md'
+      - 'LICENSE'
+      - 'Dockerfile'
+      - 'JavaHome.class'
+      - '.licenserc.yaml'
```

Comment:
Too many things to ignore. The list will definitely change often. Can we use explicit include list instead?

Reply:
Addressed and tested in dev box. Will reflect in next commit.